set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
```

Comment:
I changed it to `saveRequested`. when looking at npm code I noticed that's what they use.

```
// npm/lib/install.js
var saveRequested = require('./install/save.js').saveRequested
```